bibangamba/react-fast-food-fast
# #163720111 Set up a react app from scratch

#### What does this PR do?
Create a react app from scratch and ensure it uses webpack for bundling and serving the app, and babel for translating ES6 & JSX syntax into old school javascript that is understood by Node.

#### Description of tasks to be completed?
initialise a node project
install and configure webpack
install and configure babel
install react and react-dom

#### How can this be manually tested?
* clone the repo
* run `yarn install`
* navigate to the output URL
* it should show the `Hello fast food fast!` text on the web page

#### What are the relevant pivotal tracker stories?
#163720111

#### Screenshots
![image](https://user-images.githubusercontent.com/5712103/52267686-c15e1d00-294a-11e9-9e84-637e44937578.png)
